llm_cost_monitor/server.py

```
@@ -98,6 +102,94 @@ async def api_by_tag(hours: int = 24):
     return JSONResponse(get_cost_by_tag(hours))
 
 
+# ──────────────────────────────────────────────
+# Export endpoint
+# ──────────────────────────────────────────────
+
+@app.get("/api/export")
+async def api_export(
+    format: str = "json",
```

Ruff (0.15.9)

[error] Function argument `format` is shadowing a Python builtin

(A002)